src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/PathResolver.java

```diff
@@ -30,7 +29,7 @@
    *
    * @return Path to Google Cloud SDK or null
    */
-  public Path getCloudSdkPath() throws FileNotFoundException {
+  public Path getCloudSdkPath() {
```

Comment:
With the changes to the CloudSdk class, the clients should not need to use this class at all.